tools/cold-backup/README.md

```diff
@@ -1,26 +1,32 @@
-## Strimzi backup
+## Cold backup
 
-Bash script for offline (_cold_) backups of Kafka clusters on Kubernetes or OpenShift. 
-The script only supports a local file system, and you need to make sure to have enough free disk space in the target directory.
+Bash script for running cold backups of test or development Kafka clusters deployed on Kubernetes.
 
-If you think you don't need a backup strategy for Kafka because of its embedded data replication, then consider the impact of a misconfiguration, bug, or security breach that deletes all of your data. 
-For online (_hot_) backups, you can use storage snapshotting or stream into object storage.
+The backup procedure will stop the cluster operator and selected cluster for the entire duration of the backup.
+Run the script as a Kubernetes user with permission to work with PVC and Strimzi custom resources.
+The script only supports local file system, and you need to make sure to have enough free disk space in the target directory.
 
-Run the script as a Kubernetes user with permission to work with PVC and Strimzi custom resources. 
+Before restoring the Kafka cluster you need to make sure to have the right version of Strimzi CRDs installed.
+The restored cluster is actually the same Kafka cluster, because volumes contain the same Kafka cluster ID.
 
-The procedure will stop the Cluster Operator and selected cluster for the duration of the backup. 
-Before restoring the Kafka cluster you need to make sure to have the right version of Strimzi CRDs installed. 
-If you have a single cluster-wide Cluster Operator, then you need to scale it down manually. 
-You can run backup and restore procedures for different Kafka clusters in parallel. 
-Consumer group offsets are included, but not Kafka Connect, MirrorMaker and Kafka Bridge custom resources.
+If one or more ZooKeeper nodes are crash looping because there is no snapshot, you can apply the following patch to recover them.
```

Comment:
Is the tool not backing-up the snapshots? Or why is this a concern?

Reply:
This can sometimes happen when you backup and restore a freshly deployed clusters with very few or no data.

Reply:
This can happen in general, yes. But I do not see why to cover it here.
* The user can be affected by this even without using the backup tool - for example during a rolling update. So this is not unique to the backup tool.
* Going back to the first comment ... what is really the value of restoring a brand new cluster in test and development? (other then testing this tool of course)